src/FxCop/System.Runtime.Analyzers/Core/Performance/TestForEmptyStringsUsingStringLength.cs

```diff
@@ -1,61 +0,0 @@
-﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
-
-using Microsoft.CodeAnalysis;
-using System.Threading;
-
-namespace System.Runtime.Analyzers
-{
-    /// <summary>
-    /// CA1820: Test for empty strings using string length.
-    /// <para>
-    /// Comparing strings using the <see cref="string.Length"/> property or the <see cref="string.IsNullOrEmpty"/> method is significantly faster than using <see cref="string.Equals(string)"/>.
-    /// This is because Equals executes significantly more MSIL instructions than either IsNullOrEmpty or the number of instructions executed to retrieve the Length property value and compare it to zero.
-    /// </para>
-    /// </summary>
-    public abstract class TestForEmptyStringsUsingStringLengthAnalyzer<TLanguageKindEnum> : AbstractSyntaxNodeAnalyzer<TLanguageKindEnum>
```

Comment:
This is the change - it used to derive from AbstractSyntaxNodeAnalyzer which simply had a couple of abstract methods that Initialize and SupportedDiagnostics would call - those have been moved to this type itself now and the base type is gone.